.azuredevops/deploy-platform-bootstrap-env.yaml

```diff
@@ -85,9 +85,17 @@ extends:
         deploymentBranches:
           - 'refs/heads/main'
         azureRegions:
-          primary: 'UKSouth'          
-      - name: 'dev1'
+          primary: 'UKSouth'
+      - name: 'snd4'
+        outputTemplateChange: Skip
+        serviceConnection: AZR-ADP-SSV5
         dependsOn: 'snd1'
```

Comment:
This should probably be Validate like snd2.

Reply:
I was thinking we should deploy to an environment in DefraDev (SND1) before going to DEFRA (SND4).  That's why I did it that way.

Reply:
As discussed with Ajay, we've agreed to go with this setup and we can always change it if other disagree.  We should validate in DefraDev environment first but it should probably be SND3 instead of SND1.

SND3 -> SND4 -> DEV1 -> TST1 -> PRE1 -> PRD1

![image](https://github.com/DEFRA/adp-infrastructure-core/assets/39670555/179d209a-7d62-4cb2-8c74-e6cfa248be10)

Reply:
Updated pipeline structure of pipeline after discussing with Ajay and Praveen:

![image](https://github.com/DEFRA/adp-infrastructure-core/assets/39670555/3d76a7cd-aade-4f1e-98be-531687e76795)